guides/source/active_record_migrations.md

```diff
@@ -789,7 +789,7 @@ The `rake db:reset` task will drop the database and set it up again. This is
 functionally equivalent to `rake db:drop db:setup`.
 
 NOTE: This is not the same as running all the migrations. It will only use the
-contents of the current `schema.rb` file. If a migration can't be rolled back,
+contents of the current `schema.rb` or `structure.rb` file. If a migration can't be rolled back,
```

Comment:
@AnnaErshova can you change it to say: `db/schema.rb` and `db/structure.sql`, also note that your current change has `structure.rb`